README.md

```diff
@@ -27,6 +26,100 @@ In order to contribute, please follow the
 [Contributing Guidelines](
 https://github.com/scality/Guidelines/blob/master/CONTRIBUTING.md).
 
+## Pre-requisite
+
+To use vaultclient with an existing S3Connector deployment, Vault requires
+the use of AWS signature v4 and valid administration credentials
+in its administrative interfaces (that is, create, delete and list
+accounts, users and access keys). In order to make vaultcient use an
+administrative credential (accessKey, secretKey) pair you must first create a
+json file like this:
+
+```
+{
+    "accessKey": "<administrative access key for Vault>",
+    "secretKeyValue": "<administrative secret key for Vault>"
+}
+```
+
+For an existing deployment admin credentials need to be captured from the
+supervisor node. The admin credentials are stored in the same folder as the
+inventory file post S3Connector deployment completion. Example:
+
+```sh
+$> cat federation/env/s3config/vault/admin-clientprofile/admin1.json
+{
+    "accessKey": "W1JLTJYBDH7XT5YK5SL0",
+    "secretKeyValue": "S/=6J6tfEfg2fMMUtobXsdbhfq3wmSCUSvNRyK/e"
+}
+```
+
+There are three ways of passing the content of the file to vaultclient:
+
+1. Name the file `.vaultclient.conf` and place it in your home folder, that is: `~/.vaultclient.conf`
```

Comment:
You should set this as an unordered (bullet) list. Putting them in an ordered list implies a sequence.